src/components/AdminFilterBusinesses/AdminFilterBusinesses.jsx

```diff
@@ -0,0 +1,67 @@
+import './AdminFilterBusinesses.module.css';
+import BusinessTable from '../BusinessTable/BusinessTable.jsx';
+
+import { useBackend } from '../../contexts/BackendContext';
+import { useEffect, useState } from 'react';
+import { Box, IconButton, Tabs, TabList, Tab } from '@chakra-ui/react';
+import { ChevronRightIcon, ChevronLeftIcon } from '@chakra-ui/icons';
+
+const AdminFilterBusinesses = () => {
+  // Created the use states
+  const { backend } = useBackend();
+  //const [donationData, setDonationData] = useState([]);
+  const [businessDictionary, setBusinessDictionary] = useState([]);
+  const [currentBusinessNum, setCurrentBusinessNum] = useState(0);
+  const [currentPageNum, setCurrentPageNum] = useState(1);
+  const [currentTab, setCurrentTab] = useState('Active'); //change to all once all page is implemented
+  const [pageLimit, setPageLimit] = useState(1);
+
+  useEffect(() => {
+    const getData = async () => {
+      try {
+        loadInfo(currentTab);
+        console.log(businessDictionary);
+      } catch (error) {
+        console.error('Error fetching data:', error);
+      }
+    };
+    getData();
+  }, [currentTab, currentPageNum]);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has missing dependencies: 'businessDictionary' and 'loadInfo'. Either include them or remove the dependency array.